infrastructure-playbooks/rolling_update.yml

```diff
@@ -114,7 +114,7 @@
       systemd:
         name: ceph-mon@{{ ansible_fqdn }}
         state: stopped
-        enabled: yes
+        enabled: no
       ignore_errors: True
       when:
```

Comment:
@guits I'm also wondering why there's no `mon_use_fqdn | bool` in this condition. Any thought ?

Reply:
if I remember correctly, `mon_use_fqdn` usage was deprecated as of stable-3.2 anyway